docs/tutorials/vertical-fl.md

```diff
@@ -64,6 +64,22 @@ This can all be performed with ease using a data provisioning pipeline. To run i
 ## Model preparation for VFL
 It is an ongoing research topic on how the model can be orchestrated in VFL. We have decided to go with the most common approach by splitting it between the host and contributors, also referred to as **split learning**, this approach can be easily altered by moving layers between parties to hosting whole model on contributors while host provides only aggregation and/or activation function. We believe that this can better demonstrate capabilities of VFL on AzureML and most of the existing models can be easily split without requiring too much work.
 
+## Communication
+The nodes in the vertical federated learning need to communicate during the training to exchange intermediate outputs and gradients. Current implementation enable communication only between the host and contributors and no contributor to contributor access. The communication can happen on one of the two currently available channels: **sockets** or **Redis streams**.
+
+### Sockets
+This is easier to start with as the only **requirement here is that the nodes are interconnected using vnet**. This is also the **default option**. To create this type of communication channel just create instance of a **AMLCommSocket** class with world size, rank of the node and root run id
+
+### Redis streams
+In case it is not feasible to use vnets in your case you can fallback on using Redis. This, however, involves provisioning [Azure Cache for Redis](https://azure.microsoft.com/en-us/products/cache/). Please, make sure that you provision at least P1 Premium Instance with 6GB cache for the demos. However, if you would like to use it for your own data and architectures, feel free to scale it according to the needs for messages being sent between the nodes.
+
+Once you provision *Azure Cache for Redis* (this can be provisioned in whichever subscription you like):
+1. Go to *Access keys* and copy *Primary connection string*. 
+2. Go to your *Azure Machine Learning workspace* in Azure Portal and click on the *Key Vault* item.
```

Comment:
you might also need to add these instructions https://github.com/Azure-Samples/azure-ml-federated-learning/blob/damajercak/vertical-fl-redis/docs/real-world-examples/ccfraud.md#option-2-using-azure-ui